src/test/java/com/amihaiemil/camel/RtYamlLinesTest.java

```diff
@@ -92,4 +93,28 @@ public void returnsNestedLinesRight() {
         MatcherAssert.assertThat(iterator.next().number(), Matchers.is(5));
         MatcherAssert.assertThat(iterator.hasNext(), Matchers.is(false));
     }
+    
+    /**
+     * Test to check YamlLines are sorted correctly.
+     */
+    @Test
+    public void isSortedYamlLines() {
```

Comment:
@SherifWaly Can you rename the test to "yamlLinesAreOrdered" - ordered not sorted,, because we're not referring only to numbers, but to strings which must be alphabetically ordered.

Also, can you add letters to the test, not just digits? So it's a little more relevant :D